pipelines/fbc-builder/README.md

```diff
@@ -189,8 +201,10 @@ This pipeline is pushed as a Tekton bundle to [quay.io](https://quay.io/reposito
 |name|description|used in params (taskname:taskrefversion:taskparam)
 |---|---|---|
 |IMAGES_PROCESSED| Images processed in the task.| |
+|OCP_VERSION| OCP version derived from base image.| fbc-target-index-pruning-check:0.1:OCP_VERSION|
 |RELATED_IMAGES_DIGEST| Digest for attached json file containing related images| |
 |RELATED_IMAGE_ARTIFACT| The Trusted Artifact URI pointing to the artifact with the related images for the FBC fragment.| |
```

Comment:
@arewm How does `RELATED_IMAGE_ARTIFACT` get created?

Reply:
Latest change: the validate-fbc task was modified to create a trusted artifact from the `opm render` output. It will be pulled and used in the new check in lieu of running `opm render` again.